corehq/apps/app_manager/templates/app_manager/partials/releases_deploy_modal_sms.html

```diff
@@ -0,0 +1,39 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div class="tab-pane">
+    <div data-bind="visible: sms_{% if is_odk %}odk_{% endif %}url() && !generating_url()">
```

Comment:
not blocking, but in this and other places where this function name is generated dynamically, i think it'd be better to use `{% if is_odk %}sms_odk_url{% else %}sms_url{% endif %}`, so that way in case those functions ever get refactored, it's easy to search everywhere they are being called from

Reply:
good point, updated